ui/apps/dashboard/package.json

```diff
@@ -34,6 +34,7 @@
     "@heroicons/react": "2.0.16",
     "@inngest/components": "workspace:*",
     "@launchdarkly/node-server-sdk": "9.0.3",
+    "@next/third-parties": "14.1.3",
```

Comment:
It's a simple wrapper, but used this library as [advised in docs](https://nextjs.org/docs/app/building-your-application/optimizing/third-party-libraries) for simplicity as well. We may use other parts in following PRs.